lib/cldr.js

```diff
@@ -2091,6 +2091,43 @@ Cldr.prototype = {
 
     return weekData;
   },
+
+  extractTTSCharacterLabels(localeId) {
+    this.checkValidLocaleId(localeId);
+    const finder = this.createFinder(
+      this.getPrioritizedDocumentsForLocale(localeId, 'annotations')
+    );
+
+    const characterLabels = {};
+    finder('/ldml/annotations/annotation[@type="tts"]').forEach((node) => {
```

Comment:
I'd prefer `for ... of` to `forEach` for new code (I know there's a bunch of `forEach` left, will fix it at some point).